dev/PipsPager/PipsPager.vcxitems

```diff
@@ -29,17 +29,17 @@
   </ItemGroup>
   <ItemGroup>
     <Page Include="$(MSBuildThisFileDirectory)PipsPager.xaml">
-      <ControlsResourceVersion>Any</ControlsResourceVersion>
+      <ControlsResourcesVersion>Any</ControlsResourcesVersion>
       <Version>RS1</Version>
       <Type>DefaultStyle</Type>
     </Page>
     <Page Include="$(MSBuildThisFileDirectory)PipsPager_themeresources.xaml">
-      <ControlsResourceVersion>Any</ControlsResourceVersion>
+      <ControlsResourcesVersion>Any</ControlsResourcesVersion>
       <Version>RS1</Version>
       <Type>ThemeResources</Type>
     </Page>
     <Page Include="$(MSBuildThisFileDirectory)PipsPager_21h1_themeresources.xaml">
-      <ControlsResourceVersion>Any</ControlsResourceVersion>
+      <ControlsResourcesVersion>Any</ControlsResourcesVersion>
```

Comment:
@licanhua could you please update PipsPager again?:) I've removed the 21_h1 file by mistake and added it back in this pr

Reply:
I didn't get it. I merged with master this morning, and I should have includes [https://github.com/microsoft/microsoft-ui-xaml/commit/b710a90cdf4511e0c419b98cc545ee5f9eeb8433](https://github.com/microsoft/microsoft-ui-xaml/commit/b710a90cdf4511e0c419b98cc545ee5f9eeb8433).
What kind of change do you want from my PR?

---
In reply to: [579381590](https://github.com/microsoft/microsoft-ui-xaml/pull/4214#discussion_r579381590) [](ancestors = 579381590)

Reply:
@licanhua you're right, no change required because themeresourcve for latest and v2dot5 are the same. thanks:)